package com.byazt.qn;

import android.content.Context;
import android.widget.ScrollView;

/* JADX INFO: loaded from: classes2.dex */
@com.byazt.zqa.c(c = {0, 1, 2023, 20})
public class c extends com.byazt.xj.c<ScrollView> {
    public c(Context context) {
        super(context);
    }

    @Override // com.byazt.xj.ve
    /* JADX INFO: renamed from: b, reason: merged with bridge method [inline-methods] */
    public ScrollView c() {
        UGScrollView uGScrollView = new UGScrollView(this.tt);
        uGScrollView.c(this);
        return uGScrollView;
    }

    @Override // com.byazt.xj.c
    public com.byazt.xj.c.C0292c x() {
        return new com.byazt.dy.c.C0107c(this);
    }
}
